scripts/win-dev.bat

```diff
@@ -0,0 +1,19 @@
+@ECHO Go up to root
+cd ..
+
+@ECHO Install the prereqs
+pip install virtualenv
+virtualenv venv
+
+@ECHO Start the virtual environment
+call venv/Scripts/activate
+
+@ECHO Install dependencies in the venv
+pip install -r ./requirements/common-requirements.txt
+pip install -r ./requirements/dev-requirements.txt
```

Comment:
`dev-requirements.txt` contains all the others, so there is no need to install `common` and `test` by hand. Only dev should be enough.

Reply:
I didn't realise they had a built in hierarchy, thanks